apps/docs/content/components/CTABanner/react.mdx

```diff
@@ -236,7 +236,7 @@ render(<App />)
 | ---------- | ---------------------------------------------- | ----------- | -------- | ---------------------------------------------------------------- |
 | `children` | `ReactNode`, `ReactNode[]`                     | `undefined` | `true`   | Content to be displayed inside the `CTABanner.Heading` component |
 | `as`       | `'h1'`, `'h2'`, `'h3'`, `'h4'`, `'h5'`, `'h6'` | `'h3'`      | `false`  | Heading level                                                    |
-| `size`     | `'1'`, `'2'`, `'3'`, `'4'`, `'5'`, `'6'`       | `'2'`       | `false`  | Visual heading size, irrespective of level                       |
+| `size`     | `'1'`, `'2'`, `'3'`, `'4'`, `'5'`, `'6'`       | `'1'`       | `false`  | Visual heading size, irrespective of level                       |
```

Comment:
The actual default value is `"1"` which can be viewed here: https://github.com/primer/brand/blob/main/packages/react/src/CTABanner/CTABanner.tsx#L165